UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -284,13 +283,10 @@ private bool BreatheIn(GasMix breathGasMix, ReagentMix blood, float efficiency)
 			else if (bloodSaturation <= RelatedPart.HealthMaster.CirculatorySystem.BloodInfo.BLOOD_REAGENT_SATURATION_BAD)
 			{
 				RelatedPart.HealthMaster.HealthStateController.SetSuffocating(true);
-				if (efficiency < 0.5f)
```

Comment:
@Bod9001 was this needed for something or is it fine to remove?

Reply:
I removed it because players wouldn't gasp unless they had lung damage.

Reply:
Hmm if the brain is too damaged there shouldn't be any breathing at all, let alone gasping right? @Bod9001

Reply:
yeah